src/ontology/uberon-edit.obo

```diff
@@ -224906,6 +224906,35 @@ intersection_of: part_of UBERON:0003889 ! fallopian tube
 relationship: dc-contributor http://orcid.org/0000-0002-0819-0473
 property_value: dcterms-date 2022-10-21T08:55:11Z xsd:dateTime
 
+[Term]
+id: UBERON:8480038
+name: meningeal myeloid tissue
+def: "Lymphomyeloid tissue that consist mainly in granulopoietic cells but that also contains erythroid and lymphoid cells in different stages of maturation." [PMID:25130144]
```

Comment:
Thank you for providing a reference for your definition!

In uberon and other OBO ontologies, we prefer sources that are more focused around the definition itself, rather that experimental results papers that might embed the definition and give another sources.

Here we have:

_The meningeal tissue is a unique hemopoietic site in sturgeon
located over the medulla oblongata between the brain and cartilaginous
skull capsule, in a saddle-like manner [26,35]. The
meningeal tissue is mainly granulopoietic, with erythroid and
lymphoid cell lines in different stages of maturation and resembles
the hemopoietic bone marrow in mammals, histologically [26]._

F€ange R. Lymphoid organs in sturgeons (Acipenseridae). Vet Immunol
Immunopathol 1986;12:153e61.

Scharrer E. The histology of the meningeal myeloid tissue in the Ganoids Amia
and Lepisosteus. Anat Rec 1944;88:291e310.

I might prefer to have one of the two cited papers above as the cited reference

Reply:
@cmungall OK, I will change to one of these